content/en/docs/collector/custom-collector.md

```diff
@@ -164,6 +164,13 @@ processors:
 receivers:
   - gomod:
       go.opentelemetry.io/collector/receiver/otlpreceiver {{% version-from-registry collector-receiver-otlp %}}
+
+providers:
+  - gomod: go.opentelemetry.io/collector/confmap/provider/envprovider v1.16.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/fileprovider v1.16.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/httpprovider v0.110.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/httpsprovider v0.110.0
+  - gomod: go.opentelemetry.io/collector/confmap/provider/yamlprovider v0.110.0
```

Comment:
Shouldn't this use the `version-from-registry` thingie? It should work regardless so I am approving (confmap is 1.0 so this will work fine indefinitely now), but wondering if we should be consistent with the above

Reply:
Ook, I get what you mean though, so should I use that instead or this is good to go for merge already?

Reply:
I am fine to merge like this since it should work fine, and deal with updating this later

Reply:
Alright. Thank you for the feedback.  Much appreciated.